lib/datadog/profiling/scheduler.rb

```diff
@@ -7,36 +7,35 @@
 
 module Datadog
   module Profiling
-    # Periodically (every DEFAULT_INTERVAL_SECONDS) takes data from the `Recorder` and pushes them to all configured
-    # `Exporter`s. Runs on its own background thread.
+    # Periodically (every DEFAULT_INTERVAL_SECONDS) takes a profile from the `Exporter` and reports it using the
+    # configured transport. Runs on its own background thread.
     class Scheduler < Core::Worker
```

Comment:
Note: The responsibilities being removed from this class were moved into the `Exporter` class (but remain unchanged in behavior)